src/placeos-models/utilities/encryption.cr

```diff
@@ -67,6 +67,17 @@ module PlaceOS::Encryption
     String.new(clear_data.to_slice)
   end
 
+  def self.decrypt_for(user : Model::User, string : String, id : String, level : Level)
```

Comment:
Might be neater to move some of the logic here to sit as part of the `Level` enum.

```crystal
enum Level
  None
  Support
  Admin
  NeverDisplay

  def self.visible_to(user : Model::User)
    if user.is_admin?
      self <= Admin
    elsif user.is_support?
      self <= Support
    else
      self <= None
    end
end